### PR TITLE
Docs - README Overhaul

### PR DESCRIPTION
**Pull request by RossMurr4y**
## Description

Review / Overhaul of the README.
Remove the secondary hamlet-cli/README as the one will suffice and help prevent stale docs

## Motivation and Context

Provide consistency across the Hamlet Deploy ecosystem

## How Has This Been Tested?

n/a

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Refactor (non-breaking change which improves the structure or operation of the implementation)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Followup Actions

- [x] None

## Checklist:

- [ ] My change requires a change to the [documentation](https://github.com/hamlet-io/docs).
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] None of the above.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,30 +1,142 @@
...
+# git rebase -i HEAD~3
+# mark the fixups and save the commits
+git rebase -i HEAD~<number of commits since HEAD>
+
+# push your feature branch to your origin (the fork)
+git push --set-upstream origin feat/my-feature-branch
+
+# on Github, create a PR from your fork/feature-branch to upstream/master.
+# make sure you complete any Issue/PR templates provided.
+```
```
Comment (roleyfoley) [inline README.md:73]: Maybe this would be better in the .github repo. It has support for CONTRIBUTION file that will be made available in all repos

Comment (roleyfoley) [reply README.md:73]: See https://docs.github.com/en/github/building-a-strong-community/creating-a-default-community-health-file

**Review by roleyfoley — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,30 +1,142 @@
-# README
+# Hamlet Depoy - Executor Python
```
Comment (roleyfoley) [inline README.md:1]:
```suggestion
# Hamlet Deploy - Executor Python
```

File: README.md (hunk 1 above)
Comment (RossMurr4y) [reply README.md:73]: Will set this up and remove from the latest lot of PRs I've done that include this.

**Review by roleyfoley — APPROVED** [review]

**Pull request merged**